moped-editor/package.json

```diff
@@ -41,13 +41,15 @@
   },
   "dependencies": {
     "@apollo/client": "^3.6.9",
+    "@date-io/date-fns": "^1.3.13",
```

Comment:
just pointing out that MUI requires this `date-fns` wrapper — despite our project already having the `date-fns` package included